packages/vscode/package.json

```diff
@@ -21,7 +21,7 @@
     "vscode:prepublish:npm-install": "npm i --no-package-lock --omit=dev"
   },
   "engines": {
-    "vscode": "^1.100.0"
+    "vscode": "^1.84.0"
```

Comment:
StackBlitz Codeflow is probably based on 1.84.0.

<img width="691" alt="image" src="https://github.com/user-attachments/assets/c9099cd5-8214-4cd7-ab84-5f5d5b6ab9f5" />